tools/submission/generate_final_report.py

```diff
@@ -147,6 +148,8 @@ def main():
             "stable-diffusion-xl",
             "llama2-70b-99",
             "llama2-70b-99.9",
+            "llama2-70b-99-low-latency",
```

Comment:
I think it should be `llama2-70b-low-latency-99`. Also, I prefer `interactive` name better than `low-latency` since the target is not really in the low-latency regime. @mrmhodak what do you think?

Reply:
Changed it to `llama2-70b-low-latency-99`. We can discuss the name tomorrow and I'll make the necessary changes

Reply:
Sure, we will discuss this tomorrow.

Reply:
Updated to interactive